README.md

```diff
@@ -1,44 +1,183 @@
 [![Build Status](https://travis-ci.org/koopjs/koop-provider-sample.svg?branch=master)](https://travis-ci.org/koopjs/koop-provider-sample) [![Greenkeeper badge](https://badges.greenkeeper.io/koopjs/koop-provider-sample.svg)](https://greenkeeper.io/)
 
 
-# Koop Sample Provider
+# Koop Composite Provider
 
-This is a sample that demonstrates how to build a Koop Provider. Full documentation is provided [here](https://koopjs.github.io/docs/specs/provider/).
+Dynamic, composite aggregation of features across multiple servers with unique schemas into a single, virtual layer. 
 
-The data source in this case is the [TriMet Bus API](https://developer.trimet.org). You can see this provider in action [here](http://dcdev.maps.arcgis.com/home/item.html?id=2603e7e3f10742f78093edf8ea2adfd8#visualize).
+Combining data from different government agencies is often difficult because of varying data schemas (field attributes) and also the need to combine results into a single collection of features. For example, retrieving all of the address points from three counties into a single county layer with consitent field schema. 
 
-If you want to write your own provider, simply fork this repository or copy the contents.
+Koop Composite supports these needs through registration of a new _indicator_ with a schema, and then registering new sources for that indicator, as well as how their data schema align with the _indicator schema_. Then applications can query the virtual _indicator layer_ and Koop Composite automatically transforms any query filters to the local schemas, transforms the results into the indicator schema and combines the multiple responses into a single collection. 
```

Comment:
do we have a hub glossary? would be nice to link to definitions of indicators and schemas